apisix/plugins/jwt-auth.lua

```diff
@@ -138,14 +137,9 @@ function _M.check_schema(conf, schema_type)
     end
 
     if conf.algorithm == "RS256" or conf.algorithm == "ES256" then
```

Comment:
It looks like this validation is redundant with the json schema, see if it can be removed.